README.md

```diff
@@ -190,6 +191,7 @@ Connection response are provided via callbacks.
 ```javascript
 user.connect({
   provider: 'coinbase', // optional
+  providers: 'binance,coinbase,ethereum', // optional
```

Comment:
@oleg-vysotsky maybe it makes sense for `providers` to accept an array of strings instead (`providers: ['binance', 'coinbase', 'ethereum']`)? I mean the comma-separated string is more internal (url param) but from the SDK interface I guess an array would be more standard for a list.